diff --git a/content/public/browser/web_contents_receiver_set.h b/content/public/browser/web_contents_receiver_set.h
index 85c8875cf7f7e4b3e3cdd8606eee768923062d6e..fa7c45199f16b584639df273e514ed7187335625 100644
--- a/content/public/browser/web_contents_receiver_set.h
+++ b/content/public/browser/web_contents_receiver_set.h
@@ -33,6 +33,11 @@ namespace subresource_redirect {
 class SubresourceRedirectObserver;
 }
 
+class BraveDrmTabHelper;
+namespace brave_shields {
+class BraveShieldsWebContentsObserver;
+}
+
 namespace content {
 
 class RenderFrameHost;
@@ -105,6 +110,8 @@ class WebContentsFrameReceiverSetPassKey {
   friend class ::subresource_redirect::SubresourceRedirectObserver;
   friend class ScreenOrientationProvider;
   friend class TestFrameInterfaceBinder;
+  friend class ::BraveDrmTabHelper;
+  friend class ::brave_shields::BraveShieldsWebContentsObserver;
   FRIEND_TEST_ALL_PREFIXES(WebContentsReceiverSetBrowserTest,
                            OverrideForTesting);
 };
